pkg/basecluster/prepare.go

```diff
@@ -68,19 +74,31 @@ func Prepare(fileName string, validateOnly bool) (clusterName string, modifiedYa
 
 // -----------------------------------------------------------------------------
 
-func removeNamespaceThenEncode(obj runtime.Object, enc *yaml.Encoder) (modified bool, err error) {
+func removeNamespaceThenEncode(obj runtime.Object, enc *yaml.Encoder, casMax, casMin int) (modified bool, err error) {
```

Comment:
Now that this function does more than remove namespace, let's rename it to something like `performModsThenEncode()`, and then document here what changes the function makes.

Reply:
done.
changed to `prepareCAPIManifestThenEncode`